### PR TITLE
Remove '#noqa' from all imports in __init__.py

### PR DESCRIPTION
**Pull request by yakutovicha**

### CODE REVIEW

**Comment by yakutovicha** [comment]
Travis passed, but for some reason the info is not updated here.

File: aiidalab_widgets_base/__init__.py [hunk 1]
```diff
@@ -1,20 +1,19 @@
 """Reusable widgets for AiiDA lab applications."""
-# pylint: disable=unused-import
+# pylint: disable=unused-import,wrong-import-position
 from aiida import load_profile
 load_profile()
```
Comment (csadorf) [inline aiidalab_widgets_base/__init__.py:4]: I assume the 'wrong-import-position' is caused by this intermittent call? Could we just move it after the imports or disable the warning only for this line?

Comment (yakutovicha) [reply aiidalab_widgets_base/__init__.py:4]: No, otherwise we won't be able to load AiiDA objects

Comment (yakutovicha) [reply aiidalab_widgets_base/__init__.py:4]: But I can disable the warning for this line, this is true.

Comment (yakutovicha) [reply aiidalab_widgets_base/__init__.py:4]: Nope, didn't work. Prospector starts to complain about every subsequent import line.

Comment (csadorf) [reply aiidalab_widgets_base/__init__.py:4]: Ok, then let's go with the module-wide override.

**Pull request merged**